routes/users.js

```diff
@@ -1,8 +1,9 @@
 const express = require('express');
-const { createUser } = require('../controllers/userC');
+const { createUser, resetUserPassword } = require('../controllers/userC');
 
 const userRoute = express.Router();
 
 userRoute.post('/register', (createUser));
+userRoute.patch('/resetUserPassword/:id/:token', (resetUserPassword));
```

Comment:
```suggestion
userRoute.patch('/:id/:token', (resetUserPassword));
```

Your endpoint should not have a verb in it

Reply:
Okay, noted